### PR TITLE
feat: update landing page layout and content

### LINKED ISSUE #144: Improve front page (suggestions)
(I am just trying to make an issue after out meeting yesterday. However, it is more a comment with my suggestions on how the front page could be impoved) 

Text: First section is about "funds, finding personnel, and allocating resources" but that is not our main concern/aim. E.g. start with the next section ("aims to ease the burden of building, managing, sharing, and accessing health data") insead. It should be clear for the reader/reseacher in the first paragraph why seedcase is a good idea ("installable software [...] to better connect data collectors, researchers, clinicians, and other stakeholders to created data") and why we chose the name "seedcase".

We mention that it is aimed at health data, but in the last section, diabetes is all of a sudden introduced. There is no need to include this here (better to move it to the section "About" where Steno+DD2 is mentioned). 

Small correction: "people with people with" 

Logo: Add the logo as a bigger picture (not just the small one in the corner) in order for the reader to actually see what it is. Explain the name/logo and maybe simplify/change color?

### PR DESCRIPTION
**Pull request by signekb**
## Description

- This PR updates the layout and content of the landing page.
- The layout follows #239 
- I have rewritten the goals by putting two of the existing goals into one ("To create a framework for setting up an infrastructure for health data" and  "To develop the framework as installable software") and adding a goal of "leading by example". What do you think? 
- I have also added a "definition" of a data resource to make it a bit clearer for new users what is meant there. 
- These changes are done to make the landing page more visually appealing and to update the content that users/visitors of the webpage meet first.

## Related Issues

Closes #144 and #241 

See also Issues #...

## Reviewer Focus

This PR requires an in-depth review.

## Checklist

For all PRs that are not general documentation

- [ ] Tests accompany or reflect changes to the code
- [ ] Tests ran and passed locally
- [ ] Ran the linter and formatter
- [ ] Build has passed locally
- [ ] Relevant documentation has been updated

For general documentation:

- [X] Spell-check
- [X] Did the page(s) preview correctly on your machine without breaking
- [X] New category words (keywords) (if any) added to the code snippet file (none)

### CODE REVIEW

**Review by lwjohnst86 — APPROVED** [review]
Very nice! I've made some changes, like reduce amount of text and put the NNF logo and text into columns.

File: index.qmd [hunk 1]
```diff
@@ -82,33 +74,41 @@ able to browse and share it, to tending to projects using the data.
...
 ::: g-col-4
 ![](_extensions/seedcase-project/seedcase-theme/logos/software-product-logos.svg){fig-alt="The logos of the Seedcase software products."
 fig-align="right" width="200px"}
 :::
-:::
-:::
-:::
+:::::
+::::::
+:::::::
```
Comment (signekb) [inline index.qmd:95]: This looks a bit crazy. @lwjohnst86 was this your formatter that did this?

Comment (signekb) [reply index.qmd:95]: Same with L62 with **seven** colons?

**Comment by signekb** [comment]
Nice with the shorter texts! 🎉

File: index.qmd [hunk 2]
```diff
@@ -1,39 +1,114 @@
...
 
 <!-- TODO: List overview of purposes/principles here? -->
+
+::::::: landing-page-banner
+:::::: landing-page-banner-content
+## Software products
+
+::::: grid
+::: g-col-8
```
Comment (signekb) [inline index.qmd:67]: @lwjohnst86 Why is this without `{}`  in contrast to e.g., L29-30?

File: index.qmd (hunk 1 above)
Comment (lwjohnst86) [reply index.qmd:95]: I know... I used the same VSCode formatter we've all been using, not sure why it did that and I didn't have time to look into it. It is weird for sure... I thought it was a mistake at first, so fixed it, than ran formatter again and it did it again 😠 💢

File: index.qmd (hunk 2 above)
Comment (lwjohnst86) [reply index.qmd:67]: @signekb when only one class is provided, the formatter removes the `{}`. It's not me!! 😛

File: index.qmd (hunk 1 above)
Comment (signekb) [reply index.qmd:95]: Weird! Which formatter is this? I don't think it happens to me??

File: index.qmd (hunk 2 above)
Comment (signekb) [reply index.qmd:67]: Very weird!

**Pull request merged**